ocp_resources/resource.py

```diff
@@ -385,6 +387,15 @@ def __init__(
         self.teardown = teardown
         self.timeout = timeout
         self.delete_timeout = delete_timeout
+        self.node_selector = node_selector
+        self.node_selector_labels = node_selector_labels
+        self.body_node_selector = self._set_node_selector()
+
+    def _set_node_selector(self):
```

Comment:
This not set anything, please rename to prepare maybe? And use body in the name to have correlation between the two.